dist/redhat/scylla.spec

```diff
@@ -74,6 +74,8 @@ Summary:        The Scylla database server
 Requires:       %{product}-conf = %{version}-%{release}
 Requires:       %{product}-python3 = %{version}-%{release}
 AutoReqProv:    no
+Provides:       %{product}-tools:%{_bindir}/nodetool
+Provides:       %{product}-tools:%{_sysconfigdir}/bash_completion.d/nodetool-completion
```

Comment:
I don't understand this stanza.

Reply:
I believe it's unneeded. rpm auto-provides all files it packages.

Reply:
Although, I could be wrong. I'm fairly familiar with rpm but not an expert.

Reply:
Ok, I'll remove, but I'll wait until there is a consensus with @tchaikov.

Reply:
an alternative is to misuse `Conflicts`. but per https://docs.fedoraproject.org/en-US/packaging-guidelines/Conflicts/#_splitting_packages it's not encouraged. back when working on Ceph, we had similar use cases where we move a file from one package to another, and we ended up using `Provides`. for those who are interested: https://github.com/ceph/ceph/pull/20385 and https://github.com/ceph/ceph/pull/20401 . please note, i am not saying that it's correct, so we should follow. but it's proved to work.

Reply:
Ok. Looks like @tchaikov's knowledge is greater than mine (and I have a gap I didn't realize - I thought this was all automatic), so let's follow @tchaikov's recommendations. But be sure to test.

Reply:
So what is the bottom-line here? Should I remove these `Provides` lines?

Reply:
i'd suggest keep them. so that the package automatically provides `%{product}-tools:%{_sysconfigdir}/bash_completion.d/nodetool-completion` and `%{product}-tools:%{_bindir}/nodetool` (i.e., `scylla-tools`) will be removed when this package is installed.